argo/base/argo-wfc-configmap.yaml

```diff
@@ -1,27 +1,27 @@
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: workflow-controller-configmap
+  namespace: kubeflow
 data:
   config: |
     {
-    executorImage: argoproj/argoexec:v2.2.0,
+    executorImage: $(executorImage),
```

Comment:
Is $(executorImage) built in kustomize functionality or is this part of the KFP you are writing?

What's the point of using substitution here? The entire file is basically a substitution. If the user needs to customize this config map why don't they just customize it directly rather than doing a layer of indirection and configure parameters?